src/frontend/src/flows/manage/index.ts

```diff
@@ -217,18 +218,15 @@ const deviceListItem = (device: DeviceData) => html`
   </button>
 `;
 
-const recoveryNag = () => html`
-  <div class="warnBox">
-    <div class="warnIcon">${warningIcon}</div>
-    <div class="warnContent">
-      <div class="warnTitle">Recovery Mechanism</div>
-      <div class="warnMessage">
-        Add a recovery mechanism to help protect this Identity Anchor.
-      </div>
-      <button id="addRecovery" class="primary warnButton">Add Recovery</button>
-    </div>
-  </div>
-`;
+const recoveryNag = () =>
+  html`${warnBox({
```

Comment:
does this need to be wrapped in `html` or could you just have `warnBox`?

Reply:
Yes it works without wrapping, thank you!